src/include/target/QuadKopters_JR_2400_TX.h

```diff
@@ -1,5 +1,5 @@
 #ifndef DEVICE_NAME
-#define DEVICE_NAME "QuadKopters 24"
+#define DEVICE_NAME "QuadKopters 2G4"
```

Comment:
For consistency this should probably be `QK 2G4 TX` as everything else has `TX` or `RX` in it's device name.